docs/_Docs_ReputationMining.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reputation Mining Client
+section: Docs
+order: 6
+---
+
+The reputation mining client is usable for testing but it has limited functionality. Still, it is possible to run a single miner instance for usable reputation scores on a local test network.
+
+### Start Mining Client
+
+You can start the reputation mining client using the following command.
+
+```
+node packages/reputation-miner/bin/index.js --colonyNetworkAddress { COLONYNETWORK_ADDRESS } --minerAddress { MINER_ADDRESS }
+```
+
+The `minerAddress` in the execution above is the sixth account in `ganache-accounts.json` if running locally.
+
+The `colonyNetwork` address in the execution above is not the address outputted at contract deployment, but is the address of the Colony Network `EtherRouter`. See [Upgrades to the Colony Network](/colonynetwork/docs-the-delegate-proxy-pattern/) for more information about the EtherRouter design pattern.
+
+### Force Reputation Updates
+
+The client is set to provide a reputation update once per hour. For testing, you'll likely want to "fast-forward" your network through a few submissions to see usable reputation.
+
+You can move the network forward by an hour with the following command.
+
+```
+curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_increaseTime","params":[86400],"id": 1}' localhost:8545
+```
+
+Once you have moved the network forward by an hour, you can then mine a new block with the following command.
+
+```
+curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_mine","params":[]}' localhost:8545
+```
+
+Note that because reputation is awarded for the *previous* submission window, you will need to use the "fast-forward" command above to speed through at least 3 reputation updates before noticing a change in the miner's reputation.
+
+### Get Reputation from the Reputation Oracle
```

Comment:
It'll be useful to add another section on the Visualiser which can be very useful to understand the current reputation mining state. This work will be merged in #582